.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "python.pythonPath": "${workspaceRoot}/env/Scripts/python.exe",
+    "python.linting.pylintEnabled": true,
+    "python.envFile": "${workspaceRoot}/environments.env",
```

Comment:
By default, I think vscode supports loading a .env file. Any reason to not use the default and remove this line here?

Reply:
fixed.